charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -8,7 +8,8 @@ vCenter:
   credentialsSecret:
     name: "vsphere-cpi-creds"
     generate: true
-
+  cloudConfig:
+    generate: true
```

Comment:
Should we make the name configurable as well? Right now it is hardcoded to `vsphere-cloud-config` but if users are allowed to provide their own I think we should allow specifying the name.

Name should be respected in both the ConfigMap template, and the Deployment Pod spec.